### PR TITLE
Add Customer EDD

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

File: openapi/components/schemas/Edd/EddScoreDetails.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+type: object
+properties:
+  url:
+    type: string
+    format: uri
+    description: The url where score is taked from.
+  details:
+    type: string
+    description: The text where score is taked from.
```
Comment (adamaltman) [inline openapi/components/schemas/Edd/EddScoreDetails.yaml:9]:
```suggestion
    description: The url where score is taken from.
  details:
    type: string
    description: The text where score is taken from.
```

File: openapi/paths/customers@{id}@edd-score.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Full
+    - Risk Management
+    - Payments
```
Comment (adamaltman) [inline openapi/paths/customers@{id}@edd-score.yaml:7]:
```suggestion
```

File: openapi/paths/customers@{id}@edd-search-results.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Full
+    - Risk Management
+    - Payments
```
Comment (adamaltman) [inline openapi/paths/customers@{id}@edd-search-results.yaml:7]:
```suggestion
```

File: openapi/paths/customers@{id}@edd-score.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+    '404':
+      $ref: ../components/responses/NotFound.yaml
+put:
+  x-products:
+    - Full
+    - Risk Management
+    - Payments
```
Comment (adamaltman) [inline openapi/paths/customers@{id}@edd-score.yaml:33]:
```suggestion
```

File: openapi/paths/customers@{id}@edd-timeline.yaml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+      $ref: ../components/responses/Forbidden.yaml
+post:
+  x-products:
+    - Full
+    - Core
+  tags:
+    - Customers
+  summary: Create a EDD Timeline comment.
+  operationId: PostEddTimeline
+  x-sdk-operation-name: createTimelineComment
```
Comment (agualis) [inline openapi/paths/customers@{id}@edd-timeline.yaml:48]: We are already using this Id for `PostCustomerTimeline` operation.
Why not using `createEddTimelineComment`?

**Review by agualis — CHANGES_REQUESTED** [review]

**Review by agualis — APPROVED** [review]

**Review by justnero — APPROVED** [review]

**Review by akozhevnikov-rebilly — APPROVED** [review]

**Pull request merged**